### DISCUSSION TITLE
Consider forum.image.sc as communication channel

### DISCUSSION THREAD

**Discussion by imagejan** [PMEAL/porespy#389] [General]
Hi there,

in your [`README.md`](https://github.com/PMEAL/porespy/blob/dev/README.md#stay-informed), I read:

> It's surprizingly hard to communicate with our users

I'd like to suggest you join the image.sc forum (https://forum.image.sc) as a community partner (see also the [FAQ](https://forum.image.sc/t/frequently-asked-questions/18729)). I am sure you would be welcome : )

**Comment by jgostick** [comment]
Hi @imagejan 
Thanks very much for this suggestion.  If you'd suggested it a few months ago I would have signed up instantly, but Github has recently added the [Discussion feature](https://github.com/PMEAL/porespy/discussions) which provides a stack overflow type Q&A functionality, as well as general Discussions. I think I prefer the idea of having all the code, issues, and Q&A in one place, and we can also be sure that Github will be here for a long time.  We had considered hosting our own forum in the past (using something like Askbot) but the hassle wasn't worth it.  

Regarding the Substack mailing list, I see this as the only viable way to actually push a notification into people's mailbox.  Waiting for people to come to github (or visit image.sc) to get the news seems unreliable.  

BTW, I'm going to convert this issue into a discussion.
